contracts/colony/IColony.sol

```diff
@@ -411,32 +412,52 @@ interface IColony is ColonyDataTypes, IRecovery, IBasicMetaTransaction {
 
   /// @notice Deprecated
   /// @notice Sets the recipient on an expenditure slot. Can only be called by expenditure owner.
+  /// @dev Can only be called while expenditure is in draft state.
   /// @param _id Id of the expenditure
   /// @param _slot Slot for the recipient address
   /// @param _recipient Address of the recipient
   function setExpenditureRecipient(uint256 _id, uint256 _slot, address payable _recipient) external;
 
   /// @notice Sets the recipients in given expenditure slots. Can only be called by expenditure owner.
+  /// @dev Can only be called while expenditure is in draft state.
   /// @param _id Id of the expenditure
   /// @param _slots Array of slots to set recipients
   /// @param _recipients Addresses of the recipients
   function setExpenditureRecipients(uint256 _id, uint256[] memory _slots, address payable[] memory _recipients) external;
 
   /// @notice Deprecated
   /// @notice Set the token payout on an expenditure slot. Can only be called by expenditure owner.
+  /// @dev Can only be called while expenditure is in draft state.
   /// @param _id Id of the expenditure
   /// @param _slot Number of the slot
   /// @param _token Address of the token, `0x0` value indicates Ether
   /// @param _amount Payout amount
   function setExpenditurePayout(uint256 _id, uint256 _slot, address _token, uint256 _amount) external;
 
   /// @notice Set the token payouts in given expenditure slots. Can only be called by expenditure owner.
+  /// @dev Can only be called while expenditure is in draft state.
   /// @param _id Id of the expenditure
   /// @param _slots Array of slots to set payouts
   /// @param _token Address of the token, `0x0` value indicates Ether
   /// @param _amounts Payout amounts
   function setExpenditurePayouts(uint256 _id, uint256[] memory _slots, address _token, uint256[] memory _amounts) external;
 
+  /// @notice Set the token payouts in given expenditure slots. Can only be called by an Arbitration user.
```

Comment:
Incorrect, this function is for arbitration users (who would otherwise call `setExpenditureState`), to update values once the expenditure is locked. There's a separate version for the expenditure owners.